std/conv.d

```diff
@@ -3089,11 +3089,20 @@ Target parse(Target, Source)(ref Source p)
 }
 
 /**
-Parsing one character off a string returns the character and bumps the
-string up one position.
+Parsing one character off a range returns the first element and calls `popFront`.
+
+Params:
+    Target = the type to convert to
+    s = the lvalue of an input range
+
+Returns:
+    A character of type `Target`
+
+Throws:
+    A $(LREF ConvException) if the range is empty.
  */
 Target parse(Target, Source)(ref Source s)
-    if (isExactSomeString!Source &&
+    if (isSomeString!Source && !is(Source == enum) &&
```

Comment:
Adding another constraint looks like a behavior change, not just documentation. What does that have to do with being private?